File: WEB/HOSPITEC/src/app/models/paciente/reservacion.ts
export class Reservacion {
  idreservacion: number;
  cedulapaciente: number;
  ingreso: string;
  salida: string;
  nocama: string;

  constructor(reservacion, paciente, ingreso, salida, cama, procedimiento) {
    this.idreservacion = reservacion;
    this.cedulapaciente = paciente;
    this.ingreso = ingreso;
    this.salida = salida;
    this.nocama = cama;
  }

}

File: Hospital.Api/Hospital.Api/Models/reservacion.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace Hospital.Api.Models
{
    public class reservacion
    {
        [Key]
        public string idreservacion { get; set; }
        public string cedulapaciente { get; set; }
        public string ingreso { get; set; }
        public string salida { get; set; }
        public int nocama { get; set; }
    }
}
